migrations/1719409568000-multiuser.js

```diff
@@ -0,0 +1,53 @@
+import getAccountDb from '../src/account-db.js';
+
+export const up = async function () {
+  await getAccountDb().exec(
+    `CREATE TABLE users
+        (id TEXT PRIMARY KEY,
+        user_name TEXT, 
+        display_name TEXT,
+        enabled INTEGER,
+        master INTEGER);
```

Comment:
basically an admin is any user defined as admin role.
an master/owner (I'm renaming all to owner) is the owner of the server. this owner user cannot be deleted for safety reasons. It can be changed using the interface, but never deleted so there will be always a user with admin rights